Jenkinsfile

```diff
@@ -308,6 +309,56 @@ def buildDiffCoverage() {
     }
 }
 
+def buildPerformance() {
+  return {
+    // Select docker-cph-X.  We want docker, metal (brix) and only one executor
+    // (exclusive), if the machine changes also change REALM_BENCH_MACHID below
+    node('docker && brix && exclusive') {
+      getSourceArchive()
+
+      def gitTag = readGitTag()
+      def gitSha = readGitSha()
+
+      if (gitTag == "") {
+        setBuildName(gitSha)
```

Comment:
I don't know how you magically did that without committing, but it looks like the build worked so I committed those changes. Thanks for checking!